render/wlr_renderer.c

```diff
@@ -228,33 +227,35 @@ bool wlr_renderer_init_wl_display(struct wlr_renderer *r,
 }
 
 struct wlr_renderer *wlr_renderer_autocreate_with_drm_fd(int drm_fd) {
+	const char *name = getenv("WLR_RENDERER");
```

Comment:
Oh. I was about to merge this but then realized: all of this happens in `wlr_renderer_autocreate_with_drm_fd`. This means `WLR_RENDERER` is ignored if the backend doesn't return a DRM FD. It would be nicer to parse `WLR_RENDERER` even when the backend doesn't return a DRM FD.

But let's fix this in a separate PR, it's not that of a big deal.